websites/J/jpdb/presence.ts

```diff
@@ -0,0 +1,100 @@
+const presence = new Presence({
+		clientId: "988699263775154176",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let dueCountElem: Element, deckTitle: string, query: string, meaning: string;
+const decklist: string[] = [
+	"/novel",
+	"/vocabulary-list",
+	"/textbook",
+	"/anime",
+	"/youtube-video",
+	"/live-action",
+	"/video-game",
+	"/web-novel",
+	"/visual-novel",
+	"/audio",
+];
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "jpdb",
+		smallImageText: "jpdb.io",
+		startTimestamp: browsingTimestamp,
+	};
+	if (document.location.hostname === "jpdb.io") {
+		if (document.location.pathname.includes("/learn")) {
+			presenceData.details = "Viewing learn page";
+			dueCountElem =
+				document.querySelector('a[href="/learn"]').firstElementChild;
+			if (dueCountElem.getAttribute("style") === "color: green;")
+				presenceData.state = `New : ${dueCountElem.textContent}\n items`;
+			else presenceData.state = `Due : ${dueCountElem.textContent}\n items`;
```

Comment:
```suggestion
				presenceData.state = `New: ${dueCountElem.textContent} items`;
			else presenceData.state = `Due: ${dueCountElem.textContent} items`;
```

Reply:
i thought i already reverted the `\n` oopsies :p